sys/include/net/ng_pktdump.h

```diff
@@ -39,7 +39,7 @@ extern "C" {
  * @brief   Priority of the pktdump thread
  */
 #ifndef NG_PKTDUMP_PRIO
-#define NG_PKTDUMP_PRIO                 (PRIORITY_MIN - 1)
+#define NG_PKTDUMP_PRIO                 (PRIORITY_MAIN - 1)
```

Comment:
Can't tell right now if I experienced the same… In what manner does it break it? Does it starve out due to getchar?